lib/puppet/application/ssl.rb

```diff
@@ -80,8 +86,15 @@ def main
       raise Puppet::Error, _("An action must be specified.")
     end
 
-    Puppet.settings.use(:main, :agent)
-    host = Puppet::SSL::Host.new(options[:certname])
```

Comment:
`certname` is not a documented option and was ignored when specified on the command line.
Not passing it does the right thing.

Reply:
Let's change that to `Puppet::SSL::Host.new(Puppet[:certname])`

Reply:
Okay!